src/scss/partials/_logo.scss

```diff
@@ -42,15 +42,6 @@ $logo-color--transparent    : $white;
             height: 32px;
         }
 
-        @if ($theme == 'je') {
-            path {
-                fill: $logo-color;
-
-                .c-header--transparent & {
-                    fill: $logo-color--transparent;
```

Comment:
Will removing this not break the styles when the logo needs to be on a transparent background?

Reply:
At the moment we have colours inside svgs and separate svgs for white and transparent backgrounds

Reply:
So nothing changes without these styles, only the weird behavior of red version. The better way is to have one svg without colour and add colour via styles. We will need to ask design team for correct svg version, because at the moment in svgs there are few places with fill colour and I am not sure how to correctly remove color without breaking things. I can add a ticket for that.

Reply:
Yeh, that would be good – we'll definitely need it when we come to moving this over to the header Vue component shortly.